src/components/Multisig.tsx

```diff
@@ -762,12 +764,498 @@ function AddTransactionDialog({
+        const {mint: redeemableMint, instructions: redeemableMintItxs} = await createMint(multisigClient.provider, multisigSigner);
+        console.log("poolUxp: ");
+        const {account: poolUxp, instructions: poolUxpItxs} = await createTokenAccount(multisigClient.provider, uxpMint, multisigSigner);
+        console.log("poolUsdc: ");
+
+        const {account: poolUsdc, instructions: poolUsdcItxs} = await createTokenAccount(
+            multisigClient.provider,
+            usdcMint,
+            multisigSigner
+        );
+        itxs.push(...redeemableMintItxs, ...poolUxpItxs, ...poolUsdcItxs);
+            console.log({uxpMint})
+           // We use the uxp mint address as the seed, could use something else though.
+        const [_poolSigner] = await anchor.web3.PublicKey.findProgramAddress(
+            [uxpMint.toBuffer()],
+            UXDIDOProgramAdress
+        );
+        const poolSigner = _poolSigner;
+        const accounts = [
+            //? Do we need these two for some multisig stuff?
+              // {
+              //     pubkey: multisig,
+              //     isWritable: true,
+              //     isSigner: false,
+              // },
+              // {
+              //     pubkey: multisigSigner,
+              //     isWritable: false,
+              //     isSigner: true,
+              // },
+              // HERE NEED TO ADD THE RIGHT ACCOUNTS -- Not sure what can be hardcoded or not your call for now.
+              // Accounts expected can be found here https://github.com/UXDProtocol/uxd_ido/blob/main/programs/uxd_ido/src/lib.rs#L281
+  
+              // pool_account -- can be created arbitrarily - then will need to be referenced in the other operations and used on the front end (THE POINTER TO OUR IDO)
+              {
+                  pubkey: poolAccount.publicKey,
+                  isWritable: true,
+                  isSigner: true,
+              },
+              // pool_signer -- this is the multisig PDA : 35F3GaWyShU5N5ygYAFWDw6bGVNHnAHSe8RKzqRD2RkT
+              //? While testing we used to use a derivation from the uxp mint for creating this account
+              {
+                  pubkey: poolSigner,
+                  isWritable: false,
+                  isSigner: true,
+              },
+              // redeemable_mint --  Wrapped sollet USDC? which one do we use https://solscan.io/token/BXXkv6z8ykpG1yuvUDPgh732wzVHB69RnB9YgSYh3itW
+              // apparently we do not have to give a shit, jsute create a random mint account
+              {
+                  pubkey: redeemableMint,
+                  isWritable: false,
+                  isSigner: false,
+              },
+              // uxp_mint -- This is UXP UXPhBoR3qG4UCiGNJfV7MqhHyFqKN68g45GoYvAeL2M token () https://solscan.io/token/UXPhBoR3qG4UCiGNJfV7MqhHyFqKN68g45GoYvAeL2M ()
+              {
+                  pubkey: new PublicKey("UXPhBoR3qG4UCiGNJfV7MqhHyFqKN68g45GoYvAeL2M"),
```

Comment:
Please no hardcoded variable. How do we keep up with this. I wanna rewrite it already, feel like so easy to get mixed up

Reply:
also same as above, please prepare variable with explicit names before and assign the value in one place, there here use said variable for explicitness so we can reason